admin-dev/themes/new-theme/js/app/pages/localization/components/languages/languages-form.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <form>
+    <div class="row">
+      <div  class="col-sm-4">
+        <label>{{trans('label_currency')}}</label>
+        <PSHelp 
+          content="Vivamus sagittis lacus vel augue laoreet rutrum faucibus." 
+          placement="right"
+        />
+        <PSSelect id="select-languages" data-toggle="select2">--</PSSelect>
+      </div>
+      <div  class="col-sm-2">
+        <label class="d-block">{{trans('label_switch')}}</label>
+        <PSSwitch />
+      </div>
+    </div>
+    <div class="row align-items-end mt-4">
+      <div  class="col-sm-5">
+        <label>{{trans('label_language_name')}}</label>
+        <PSInput />
+      </div>
+      <div class="col-sm-1">
+        <label class="d-block">
+          {{trans('label_switch_rtl')}}
+          <PSHelp 
+            content="Vivamus sagittis lacus vel augue laoreet rutrum faucibus." 
```

Comment:
Is it intended to stay filled with lorem ipsum data ? (I'm spamming. This is the last comment about Lorem Ipsum)